Jenkinsfile

```diff
@@ -116,7 +116,7 @@ pipeline {
                 }
             }
             steps{
-                withMaven(maven: 'Maven 3.5.2', jdk: 'jdk8-latest', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings', mavenOpts: '${LINUX_MVN_RANDOM}') {
+                withMaven(maven: 'maven-latest', jdk: 'jdk8-latest', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings', mavenOpts: '${LINUX_MVN_RANDOM}') {
```

Comment:
There might be negative impacts during the deploy stage (i.e. if maven-deploy-plugin needs to be pinned/upgraded). I'm not sure we will really know, until we merge this PR though.